packages/block-library/src/cover/index.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Server-side rendering of the `core/categories` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/categories` block on server.
+ *
+ * @param array $attributes The block attributes.
+ *
+ * @return string Returns the categories list/dropdown markup.
+ */
+function render_block_core_cover( $attributes, $content ) {
+	if( $attributes['useFeaturedImage'] === false ) {
+		return $content;
+	}
+
+	$currentFeaturedImage = get_the_post_thumbnail_url();
```

Comment:
This can return `false`. We need a fallback or simply not output in this case.

https://developer.wordpress.org/reference/functions/get_the_post_thumbnail_url/